test/e2e/BYOHDockerFile

```diff
@@ -9,6 +9,7 @@ RUN apt-get update \
        rsync nfs-common fuse-overlayfs containerd curl gnupg2\
     && mkdir -p /etc/containerd \
     && containerd config default | tee /etc/containerd/config.toml \
+    && sed 's/SystemdCgroup = false/SystemdCgroup = true/'  /etc/containerd/config.toml \
```

Comment:
It's good to add a little bit of context in the PR or issue about why we are moving from `cgroupfs` to `SystemdCgroup`

Reply:
Yea. I was just trying out a few things. Closing it now. Will open a new one later.